ledger/light/proc/hot_data.go

```diff
@@ -156,10 +155,16 @@ func (p *HotObjects) notifyPending(
 		return
 	}
 
-	_, err := p.Dep.MessageBus.Send(ctx, &message.AbandonedRequestsNotification{
-		Object: objectID,
-	}, nil)
+	msg, err := payload.NewMessage(&payload.AbandonedRequestsNotification{
+		ObjectID: objectID,
+	})
 	if err != nil {
-		inslogger.FromContext(ctx).Error("failed to notify about pending requests")
+		inslogger.FromContext(ctx).Error("failed to create reply")
```

Comment:
why this text was changed? also original error is thrown away.

Reply:
Is it ok? `"failed to notify about pending requests: failed to create reply"`